packages/app/obojobo-document-engine/src/scripts/common/text/styleable-text-renderer.js

```diff
@@ -159,8 +159,13 @@ const wrapElement = function(styleRange, nodeToWrap, text) {
 			)
 			nodeToWrap.parent.replaceChild(nodeToWrap, newChild)
 			newChild.addChild(nodeToWrap)
-			const html = window.katex.renderToString(text, { throwOnError: false })
-			nodeToWrap.html = `<span aria-hidden="true">${html}</span>`
+
+			try {
+				const html = window.katex.renderToString(text, { throwOnError: false })
+				nodeToWrap.html = `<span aria-hidden="true">${html}</span>`
+			} catch (e) {
+				console.error(e) // eslint-disable-line
+			}
```

Comment:
```suggestion
			// if we're running in the browser, render katex
			// note: this does get called on the server, but we don't need the rendered katex there
			if(typeof window !== "undefined"){
				const html = window.katex.renderToString(text, { throwOnError: false })
				nodeToWrap.html = `<span aria-hidden="true">${html}</span>`
			}
```

Some observations: 

- prefer not to console.error here, especially when it's expected to throw errors for the server (keep logs lighter, free of miss-reported errors)
- this would be more useful as an if - it'll self document a little better (if isBrowser do ...)

Reply:
I like this implementation